packages/terra-app-delegate/docs/README.md

```diff
@@ -145,3 +134,7 @@ ExampleRootComponent.propTypes = {
 }
 
 ```
+
+### Deprecated APIs
+
+The Disclose API for specifying which component to present was previously structured to support Redux-based disclosure managers. However, the disclosure managers were transitioned away from Redux to allow consumers to have greater control over the disclosure lifecycle. Therefore, the `name` and `props` keys in the disclose API are now deprecated, and the `component` key should be used to provide a component instance to present instead. This also means that the `registerComponentForDisclosure`/`getComponentForDisclosure` are deprecated as well, as there is no need for component retreival when an instance is given through the API. Those APIs will continue to function, but the usage of the `component` key is recommended.
```

Comment:
> Therefore, the `name` and `props` keys in the disclose API are now deprecated, and the `component` key should be used to provide a component instance to present instead.

The name and props approach works better than the component approach for ReactOnRails rendering...although I don't think that should be a dealbreaker as I'm fine with a slightly different API for the hybrid disclosures (we already need to know whether an application or container class should be disclosed).

+1 on deprecated registerComponentForDisclosure/getComponentForDisclosure though.